grafana/provider_test.go

```diff
@@ -74,3 +73,14 @@ func testAccPreCheck(t *testing.T) {
 		}
 	})
 }
+
+// testAccExample returns an example config from the examples directory.
+// Examples are used for both documentation and acceptance tests.
+func testAccExample(t *testing.T, path string) string {
+	path = fmt.Sprintf("../examples/%s", path)
+	example, err := ioutil.ReadFile(path)
```

Comment:
Nit: 
```suggestion
	example, err := ioutil.ReadFile("../examples/" + path)
```

There's no actual need for `fmt.Sprintf` here, and it will definitely be much faster:

```
BenchmarkCreateSimpleString/Sprintf-12         	14443981	        79.73 ns/op	      24 B/op	       1 allocs/op
BenchmarkCreateSimpleString/plus-12            	1000000000	         0.6006 ns/op	       0 B/op	       0 allocs/op
```